Project/src/MakeCall/AddParticipantPopover.js

```diff
@@ -1,17 +1,22 @@
 import React, { useState, useRef } from "react";
 import { TextField } from 'office-ui-fabric-react';
 import { Button } from 'office-ui-fabric-react'
-
+import { CallKind } from "@azure/communication-calling";
+import { utils } from '../Utils/Utils';
 
 export default function AddParticipantPopover(props) {
```

Comment:
nit: you could destructure the props here so you avoid using `props.xyz`
```suggestion
export default function AddParticipantPopover({ call }) {
```